.tool-versions

```diff
@@ -1 +1,2 @@
 terraform 1.9.8
+nodejs 22.5.0
```

Comment:
This could be removed if `.node-version` can use a SemVer `2.5.0`. ASDF complained otherwise.